packages/mongo-async/collection.js

```diff
@@ -745,16 +745,16 @@ Object.assign(Mongo.Collection.prototype, {
 
   // We'll actually design an index API later. For now, we just pass through to
   // Mongo's, but make it synchronous.
-  _ensureIndex(index, options) {
+  async _ensureIndex(index, options) {
```

Comment:
When is this going to be released? `_ensureIndex`, `createIndex`, and others are all part of the public collections API.

If it's going to be 3.0, maybe we should already remove the underscored methods as well as the deprecated ones and simply expose fibers-less ones, just like I did in #12295?

Reply:
> When is this going to be released? _ensureIndex, createIndex, and others are all part of the public collections API.

It is going to 3.0. Will try my best to always add this info to the pr's so everyone is always on the same page :) 

I'm not sure what do you mean by ``maybe we should already remove the underscored methods as well as the deprecated ones and simply expose fibers-less ones``  because it is already a wrapper for the raw mongo collection(with a little bit more like logging). 
I will make a commit with what I think it is

Reply:
I mean having a clear API - no underscores and no two methods doing the same thing (`_ensureIndex` and `createIndex`).